.travis.yml

```diff
@@ -23,7 +23,7 @@ addons:
       - g++-4.8
 before_install:
   - export CXX="g++-4.8" CC="gcc-4.8"
-  - nvm install 6
+  - nvm install 8
```

Comment:
ReSpec will track Node lts starting 31st of October, so please be sure to change this to lts to avoid issues in the future.

Reply:
At the moment LTS seems to be 6.11.4 - I assume ReSpec won't track it until it moves to 8?

Reply:
Yep, waiting till 8. Just 1 week away.